keyvault-diskencryption-lab/New-EncryptedVM.ps1

```diff
@@ -116,17 +68,42 @@ Write-Host "Building the VM: $VMName..."
 New-AzVM -ResourceGroupName $ResourceGroupName -Location $Location -VM $VirtualMachine
 #endregion 
 
+#region KeyVault
```

Comment:
I assume the keyvault is moved down because the key vault lab step is after the VM lab step?

Reply:
I just thought it flowed better and aligns with the official documentation. I will be updating the labstep content and order before merging.